tools/incubator.js

```diff
@@ -103,6 +103,11 @@ const Incubator = new Lang.Class({
         basin_metadata['lastModifiedDate'] = metadata['lastModifiedDate'];
         basin_metadata['indexed'] = false;
 
+        /* Needs to be set by libingester at ingestion time, otherwise
```

Comment:
The way I was thinking about this, is that it would be the exact inverse of the `resources` property. Since resources are determined in Basin, it wouldn't necessarily need to be set by libingester. Could be implemented as an extra pass over the manifest before outputting it.

Reply:
I suppose we would do that in `_import_hatch`, after everything has been imported.

It'd be O(N^2), but I guess that's not too bad.